bin/watch-packages-and-rebuild.js

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env node
+const path = require( 'path' );
+const chokidar = require( 'chokidar' );
+const util = require( 'util' );
+const promiseExecFile = util.promisify( require( 'child_process' ).execFile );
+const debounce = require( 'lodash/debounce' );
+const debouncedProcessRebuildQueue = debounce( processRebuildQueue, 100 );
+
+const packagesDirectoryPath = path.join( '.', 'packages' );
+const watcher = chokidar.watch( packagesDirectoryPath, {
+	ignored: /^\./,
+	persistent: true,
+	ignoreInitial: true,
+} );
+watcher.on( 'change', handleChange ).on( 'add', handleChange ).on( 'unlink', handleChange );
+
+const rebuildQueue = [];
+
+async function processRebuildQueue() {
+	const queueSnapshot = [ ...rebuildQueue ];
+	rebuildQueue.length = 0;
+	await Promise.all( queueSnapshot.map( rebuildPackage ) );
+}
+
+function handleChange( filePath ) {
+	if ( isPathIgnored( filePath ) ) {
+		return;
+	}
+	console.log( 'heard change in', filePath );
```

Comment:
This is optional but it would really make our lives easier:

```suggestion
	console.log( chalk.cyan('heard change in'), filePath );
```

`chalk` is already installed. I'd like this because then I won't have to read, seeing the color is enough.

Reply:
Great idea! I forgot about chalk.